omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/visit/VisitDatesFragmentController.java

```diff
@@ -40,10 +40,10 @@ public SimpleObject setDuration(@SpringBean("visitService") VisitService visitSe
                                             @RequestParam(value="stopDate", required = false) Date stopDate,
                                             HttpServletRequest request, UiUtils ui) {
 
-        boolean allowChangingVisitTime = BooleanUtils.toBoolean(Context.getAdministrationService().getGlobalProperty(CoreAppsConstants.GP_ALLOW_CHANGING_VISIT_TIME));
+        boolean allowConvertingTimezones = BooleanUtils.toBoolean(Context.getAdministrationService().getGlobalProperty(UiFrameworkConstants.GP_TIMEZONE_CONVERSIONS));
```

Comment:
@icrc-toliveira i find the name of `GP_ALLOW_CHANGING_VISIT_TIME` easier to understand than what you changed it to.

Secondly, do you mind explaining how changing the used global property fixes the reported bug?